electron/achievements.js

```diff
@@ -2,16 +2,22 @@
 const greenworks = require("./greenworks");
 const log = require("electron-log");
 
-function enableAchievementsInterval(window) {
+async function enableAchievementsInterval(window) {
    // This is backward but the game fills in an array called `document.achievements` and we retrieve it from
   // here. Hey if it works it works.
   const steamAchievements = greenworks.getAchievementNames();
+  log.info(`All Steam achievements ${JSON.stringify(steamAchievements)}`);
```

Comment:
These logs should probably be log.verbose (or is it debug..).  The output looks to be pretty large.

Reply:
They are written once per game process, but the lines are fairly long at 1.5kB for the all achievements listing. I can drop the level to debug.